rust/stracciatella/src/schemas/yaml/game.schema.yaml

```diff
@@ -48,15 +48,18 @@ properties:
       power consumption, higher values can make the game appear to be more responsive.
 
       Stracciatella default value: `40`
-  ms_per_time_slice:
```

Comment:
Will this break upgraded installs, requiring user intervention? I don't remember when all we check against the schema, but if it's also at runtime, it'd be better to leave this entry in and mark it as deprecated instead.

Reply:
I didn't know the answer, so I just tried it: the validation checks whether all the properties marked as required are present and whether properties have the correct type within the allowed range. Additional properties are ignored, so having an old and unused value for ms_per_time_slice won't cause an error when the game starts.